packages/block-editor/src/components/url-input/style.scss

```diff
@@ -120,7 +120,7 @@ $input-size: 300px;
 }
 
 .block-editor-url-input__button-modal {
-	box-shadow: $shadow-popover;
+	box-shadow: $elevation-x-small;
 	border: 1px solid $gray-300;
```

Comment:
Not a blocker, just noticing. Should this be $border-width instead of 1px?

Reply:
Yes, there a number of instances of such oversights 😓 

Let's open a separate PR to address, hopefully it's a quick find & replace for this one.